src/Microsoft.ComponentDetection.Detectors/npm/Contracts/PackageLockV1.cs

```diff
@@ -1,4 +1,4 @@
-﻿#nullable enable
+#nullable enable
```

Comment:
Not super relevant to this pull request, but is there a reason that we don't have nullable configured at our project level with `<Nullable>enable</Nullable>`?

Reply:
We probably could - I'm not aware of any reasoning behind why we wouldn't allow it, but agree that this is out of scope for this PR